### PR TITLE
 update imports in harvest.api

### PR DESCRIPTION
**Pull request by victorphoenix3**
fixes: #68

Signed-off-by: Jayati Shrivastava <gaurijove@gmail.com>

### CODE REVIEW

**Comment by tristanlatr** [comment]
Thanks for your contribution!
Please note that util PR #114 is merged, all new pull requests should target the ``develop`` branch. 
I'll let you change the target branch, since this kind of changes will likely create conflicts.
Sorry about that.

**Comment by victorphoenix3** [comment]
@tristanlatr Hi, I rebased my changes onto the top of develop and have changed the base of the PR.

**Comment by tristanlatr** [comment]
Thanks, if I read the diff correctly, there were no more start imports actually.

**Comment by tristanlatr** [comment]
Hi @victorphoenix3,

I think your patch is adding unused imports. I think this is due to the rebasing.

Could you fix it?

Thanks

**Comment by victorphoenix3** [comment]
@tristanlatr looks like the star imports are only present in the master branch, and not in the develop branch. I have removed all the unused imports from `harvest/api.py`.

File: saskatoon/harvest/api.py [hunk 1]
```diff
@@ -3,13 +3,18 @@
 from django.urls import reverse, reverse_lazy
 from rest_framework import viewsets
 from rest_framework.response import Response
+from rest_framework import viewsets
```
Comment (tristanlatr) [inline saskatoon/harvest/api.py:6]: Looks like this one is already imported at line 4.
```suggestion
```

Comment (victorphoenix3) [reply saskatoon/harvest/api.py:5]: sorry. missed. 😐

File: saskatoon/harvest/api.py [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+    OrganizationFilter, EquipmentFilter )
+from harvest.forms import ( RequestForm, CommentForm, RFPManageForm, 
+    HarvestYieldForm )
+
 from member.models import AuthUser, Organization
-from .models import Equipment, Harvest, HarvestYield, Property, RequestForParticipation, Comment
-from .serializers import ( HarvestSerializer, PropertySerializer, EquipmentSerializer,
+from harvest.models import ( Harvest, Property, HarvestYield, Comment,
+    Equipment, RequestForParticipation)
+from harvest.serializers import ( HarvestSerializer, PropertySerializer, EquipmentSerializer,
```
Comment (2ynn) [inline saskatoon/harvest/api.py:16]: @victorphoenix3 I'm sorry but I think you need to rebase your branch onto master. I've already pushed similar changes.

File: saskatoon/harvest/serializers.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 from rest_framework import serializers
-from .models import Harvest, Property, Equipment, EquipmentType, RequestForParticipation
+from harvest.models import Harvest, Property, Equipment, EquipmentType, RequestForParticipation
```
Comment (2ynn) [inline saskatoon/harvest/serializers.py:2]: I think this will be the only change left

File: saskatoon/harvest/api.py (hunk 2 above)
Comment (victorphoenix3) [reply saskatoon/harvest/api.py]: Done

File: saskatoon/harvest/serializers.py (hunk 3 above)
Comment (victorphoenix3) [reply saskatoon/harvest/serializers.py:2]: Yes

**Comment by 2ynn** [comment]
@victorphoenix3 thanks for rebasing and sorry for the confusion about branches

**Review by 2ynn — APPROVED** [review]

**Pull request merged**